website/docs/07-apps/01-included-in-onion/pico8-wrapper.md

```diff
@@ -0,0 +1,115 @@
+---
+
+:::caution 
+This section refers to an application planned for inclusion in a future Onion release. Final instructions and usage details may differ from the ones described here. 
+Check [here](https://www.reddit.com/r/MiyooMini/comments/17nw79r/announcing_native_pico8_wrapper_for_onion_with/k7ueyec/?context=3) for playtesting details.
+:::
+
+# PICO-8 Native Wrapper / Splore
+
+The PICO-8 Native Wrapper allows you to play PICO-8 without the need for emulation (hence providing full compatibility with all PICO-8 games), and grants access to Splore, a "frontend" app that allows you to browse, search and play through the entirety of the PICO-8 catalogue (as indexed by the [Lexaloffle BBS](https://www.lexaloffle.com/bbs/)), as long as you have an internet connection. 
+
+Splore also allows you to navigate through and play local cartridge files.
+
+This is an alternative to the RetroArch `fake-08` core, which has access to all typical RetroArch features such as rewinding, fast forward and save state management, but can have compatibility with a portion of the PICO-8 library, specially recently released games.
+
+## How to Enable and Launch
+
+1. Enable the `PICO-8 Native Wrapper` App using the Onion Package Manager's *Expert* section.
+
+2. In order to use the wrapper, you need to have previously purchased PICO-8. It is available as a single purchase at [Lexaloffle's website](https://www.lexaloffle.com/games.php?page=updates). Once downloaded, place the `pico8_dyn` and `pico8.dat` files from the **Raspberry Pi** package in the `/App/pico/bin` directory of your Onion SD card.
+
+:::tip
+Do not use the FTP wireless transfer service to place these files on the SD card, as it may strip their execution privileges and cause the application to fail to launch.
+:::
+
+3. Afterwards, launch Splore from the PICO-8 application under the Onion *Apps* submenu.
+
+## Basic Usage
+
+Using Splore you can browse, search and play through cartridges listed on the Lexaloffle PICO-8 BBS. Use the <kbd>LEFT</kbd> and <kbd>RIGHT</kbd> buttons to switch through the following sections:
+
+* **Favorites**: Lists the games you have previously marked as favorites.
+* **New**: Shows the most recently indexed cartridges on the Lexaloffle BBS.
+* **Featured**: Shows a list of curated and highly rated cartridges by the Lexaloffle team.
+* **Work in Progress**: Incomplete cartridges currently in development.
+* **Jam**: Cartridges developed for game jam events hosted regularly by Lexaloffle.
+* **Lucky Draw**: A random game selection.
+* **Search**: Allows you to search for cartridges based on their names and description. Use the the up/down buttons to select an alphanumeric character and the right/left buttons to move between slots.
+* **/ (File Browser)**: Allows you to browse the local filesystem. You can access the cache for previously launched Splore games (`BBS`) or the cartridges located in your `/Roms/PICO` directory (`carts`).

```

Comment:
For this and the other reference, we redirected the BBS save location to /Roms/PICO/splore (so any carts downloaded through splore will go here) - When you look in the local filebrowser in the carousel within Pico you'll see this new splore dir.